tools/test.js

```diff
@@ -262,6 +265,155 @@ var testResults = Promise.join(tests, buildResult, function(tests) {
                 }
             })
         }).then(function() {
+            // This is very initial version of performance regression test
+            // This test makes an assumption that local git repository does not have uncommitted changes
+            if (!(process.env.TRAVIS || process.env.CI || options.testPerformance)) {
```

Comment:
Only testPerformance flag should test it, I will bake it in release scripts instead of running it in CI. That way results are much more stable as well.